tap_zoom/client.py

```diff
@@ -90,7 +100,7 @@ def request(self,
         if url is None and \
             self.__use_jwt == False and \
             (self.__access_token is None or \
-             self.__expires_at <= datetime.utcnow()):
+             self.__expires_at <= now()):
```

Comment:
what is the significance of having `__expires_at < now()` check here ?

Reply:
Actually, it doesn't add any value but it was there already and hence, I kept it as it is.

Reply:
If it doesn't add any value then what is purpose of adding this [line](https://github.com/singer-io/tap-zoom/blob/c2951aaae14c26ad3777d8c5746b55091418a38d/tap_zoom/client.py#L37) ?

Reply:
I was talking about the changes done from datetime.utcnow() to now() which actually doesn't change anything, it is just that instead of using inbuilt datetime, we're using singer-python utility.
If you're asking about this `__expires_at <= now()` overall then it is firstly checking if we have access_token availbale or not and if yes then it is expired or not?

Reply:
Let me very clear 
in the master branch we have __expires_at at two places [here](https://github.com/singer-io/tap-zoom/blob/master/tap_zoom/client.py#L61) and [here](https://github.com/singer-io/tap-zoom/blob/master/tap_zoom/client.py#L93) but in this feature branch you've also declared the same attribute (`__expires_at`) in the __init__ method [here](https://github.com/singer-io/tap-zoom/blob/TDL-22604-dev-mode-changes/tap_zoom/client.py#L37). So, my question is, why was it declared again in the __init__ method ?

Reply:
I have defined one dummy value to the **___expires_at_** variable in init for dev mode implementation. Because, when **dev_mode = False**, it initialises the value of **_self.__expires_at_**  on line no. 72 and hence, condition on line no. 103 gets evaluated properly.
But if I don't declare the __expires_at attribute and executes the code with **dev_mode=True**, it throws below error,
```AttributeError: 'ZoomClient' object has no attribute '_ZoomClient__expires_at'```